Sprint-3/1-key-implement/3-get-card-value.js

```diff
@@ -8,9 +8,22 @@
 // write one test at a time, and make it pass, build your solution up methodically
 // just make one change at a time -- don't rush -- programmers are deep and careful thinkers
 function getCardValue(card) {
-    if (rank === "A") return 11;
+  const rank = card.slice(0, -1);
+  if (rank === "A") return 11;
+  if (rank === "2") return 2;
+  if (rank === "3") return 3;
+  if (rank === "4") return 4;
+  if (rank === "5") return 5;
+  if (rank === "6") return 6;
+  if (rank === "7") return 7;
+  if (rank === "8") return 8;
+  if (rank === "9") return 9;
+  if(["10", "J", "Q", "K"].includes(rank)) return 10;
+  throw new Error("Invaild card rank")
 }
```

Comment:
This works. You might want to consider if `if` is the cleanest way of doing this. JavaScript has another control flow statement known as a `switch` block which may be more elegant if you have many conditions like above (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch). Either works.